### DISCUSSION TITLE
怎么关闭绿色的LED灯?

### DISCUSSION THREAD

**Discussion by xielucky** [coolsnowwolf/lede#9526] [R68S / 电犀牛专区]
蓝色的可以关,但绿色的没选择呢!
![image](https://user-images.githubusercontent.com/25549793/173195670-59a6ddc9-5b46-4376-a490-797d1f7ace69.png)

**Comment by aiamadeus** [comment]
这个关不了，是电源直接驱动的。

**Reply by kiong2016** [reply]
我想知道能不能关闭网卡灯

**Reply by smomop** [reply]
我也想知道
